lib/rules/comment-pattern/__tests__/index.js

```diff
@@ -0,0 +1,113 @@
+'use strict';
+
+const { messages, ruleName } = require('..');
+
+testRule({
+	ruleName,
+	config: [/foo-.+/],
+
+	accept: [
+		{
+			code: '/* foo-valid yay */',
+		},
+		{
+			code: `/* foo--
+			multi-line
+			\n
+			\r
+			\r\n
+			\n\r
+			\t
+				*/`,
+		},
+		{
```

Comment:
We can remove these `.without-comment {}` tests as they are duplicated in the [basic checks](https://github.com/stylelint/stylelint/blob/8366b54f536b26c2a994865568336c577ea5625e/lib/testUtils/basicChecks.js).